docs/validators/notaryv1.md

```diff
@@ -0,0 +1,210 @@
+- name: docker_essentials
+  type: notaryv1
+  host: notary.docker.io
+  trust_roots:
+  - name: sse
+    key: |
+      -----BEGIN PUBLIC KEY-----
+      MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni
+      qiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==
+      -----END PUBLIC KEY-----
+
+policy:
+- pattern: "docker.io/securesystemsengineering/connaisseur:*"
+  validator: docker_essentials
+  with:
+    key: sse
+    delegations:
+    - belitzphilipp
+    - starkteetje
+```
+
+## Additional Features
+
+### Delegations
+
+Notary (V1) offers the functionality to delegate trust. To better understand this feature, it's best to have a basic understanding of the TUF key hierarchy, or more specifically the purpose of the root, targets and delegation keys. If you are more interested in this topic, please read the [TUF documentation](https://theupdateframework.github.io/specification/latest/#roles-and-pki).
+
+When creating the signatures of your docker images earlier, two keys were generated -- the root key and the targets key. The root key is the root of all trust and will be used whenever a new image repository is created and needs to be signed. It's also used to rotate all other kinds of keys, thus there is usually only one root key present. The targets key is needed for new signatures on one specific image repository, hence every image repository has its own targets key. Hierarchically speaking, the targets keys are below the root key, as the root key can be used to rotate the targets keys should they get compromised.
+
+Delegations will now go one level deeper, meaning they can be used to sign individual image repositories and only need the targets key for rotation purposes, instead of the root key. Also delegation keys are not bound to individual image repositories, so they can be reused multiple times over different image repositories. So in a sense they can be understood as keys for individual signers. 
+
+To create a delegation key run:
+
+```bash
+> docker trust key generate <your-name>
+Generating key for <your-name>...
+Enter passphrase for new <your-name> key with ID 9deed25: 
+Repeat passphrase for new <your-name> key with ID 9deed25: 
+Successfully generated and loaded private key. Corresponding public key available: /home/ubuntu/Documents/mytrustdir/<your-name>.pub
+```
+
+This delegation key now needs to be added as a signer to a respective image repository, like the `busybox` one from above. In doing so, you'll be ask for the targets key.
+
+```bash
+> docker trust signer add --key <your-name>.pub <your-name> <your-private-repo>/busybox
+
+Adding signer "<your-name>" to <your-private-repo>/busybox...
+Enter passphrase for repository key with ID b0014f8: 
+Successfully added signer: <your-name> to <your-private-repo>/busybox
+```
+
+If you create a new signature for the image now, you'll be ask for your delegation key instead of the targets key, therefore creating a signature using the delegation.
```

Comment:
```suggestion
If you create a new signature for the image, you'll be asked for your delegation key instead of the targets key, therefore creating a signature using the delegation.
```